data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,72 @@ mission "Station Exploration Abandon Ship"
 			label fine
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
+mission "Pilgrimage"
+	name "Hai spiritual leader to <destination>"
+	description "Bring a Hai spiritual leader to <destination>."
+	minor
+	source
+		government "Hai"
+		not attributes "station" "unfettered"
+	destination
+		attributes "unfettered"
+	passengers 1
+	to offer
+		random < 30
+		has "outfit: Quantum Keystone"
+		has "First Contact: Hai: offered"
+	on offer
+		conversation
+			`It is an unusually foggy day on <source>. The air is so thick with moisture that you cannot see more than a few paces in front of you. Perhaps that is why you are so startled when you turn around and see a Hai standing right next to you. He is so close you can see dewdrops on his whiskers, and you immediately notice the prominent jewelry that he is wearing.`
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise. Are you willing and able to take me to <destination>? I can pay <payment>." He appears quite old, judging by his stooped-over posture and the white fur around his muzzle.`
+			choice
+				`	"I can take you there."`
+					goto agree
+				`	"I'm not going there. You'll have to ask someone else."`
+					decline
+				`	"Why would you want to go to <destination>? Isn't that an Unfettered planet?"`
+			`	The Hai pauses for a moment, as if unsure of what to say. "My teachings are more... appreciated among our northern brethren."`
+			choice
+				`	"Teachings?"`
+					goto teachings
+				`	"Alright, I'll take you there."`
+					goto agree
+				`	"Unfettered space is too dangerous for me. You'll have to find someone else."`
+					goto dangerous
+			label teachings
+			`	An unreadable expression appears on the Hai's face. "How do I explain this... You have your preachers and wise men among the humans, yes? I am something similar." He stops for a moment, realizing something. "Ah, apologies for my rudeness - you probably do not know who I am. My name is Hareek. Now, will you take me to <destination>?"`
```

Comment:
You can't only introduce Hareek by name in this conversation branch, because you use his name extensively in the `on complete` and `on visit` blocks. He should introduce himself no matter what route the player takes to the `accept`.

Reply:
I put his name in the intro to this mission, so this should be fixed now.